zipkin-autoconfigure/metrics-prometheus/pom.xml

```diff
@@ -27,6 +27,7 @@
 
     <properties>
         <main.basedir>${project.basedir}/../..</main.basedir>
+        <prometheus_simpleclient.version>0.0.23</prometheus_simpleclient.version>
```

Comment:
the fix to the aforementioned bug is in master now, so guessing 0.0.24 will sort it out